File: Java/me.neonix.versionmanager/versionmanager/Project.java
public abstract class Project {
    private String author;
    private String name;
    private Version version;

    public Project(String author, String name, Version version) {
        this.author = author;
        this.name = name;
        this.version = version;
    }

    @Override
    public String toString() {
        return "Project[" + this.author + ", " + this.name + ", " + this.version + "]";
    }

    public void setVersion(Version version) {
        this.version = version;
    }

    public Version getVersion() {
        return this.version;
    }
}

File: Python/me.neonix.versionmanager/versionmanager/project.py
class Project:
    '''
    Base class for a project.
    '''
    __slots__ = ['__author', '__name', '__version']

    def __init__(self, author, name, version):
        self.__author = author
        self.__name = name
        self.__version = version


    def __repr__(self):
        return f'Project[{self.__author}, {self.__name}, {self.__version}]'


    def set_version(self, version):
        self.__version = version


    def verison(self):
        return self.__version
